pkg/ibmcsidriver/controller_helper.go

```diff
@@ -446,10 +457,26 @@ func createCSIVolumeResponse(vol provider.Volume, capBytes int64, zones []string
 			VolumeId:           vol.VolumeID,
 			VolumeContext:      labels,
 			AccessibleTopology: []*csi.Topology{topology},
+			ContentSource:      src,
```

Comment:
In case of normal volume creation if this value is nil ? will it be impacting the flow ?

Reply:
yes. It will be nil and it doesn't impact the flow